KarhooUISDK/Screens/QuoteList/QuoteListPresenter.swift

```diff
@@ -97,9 +99,23 @@ final class KarhooQuoteListPresenter: QuoteListPresenter {
     }
 
     func getNumberOfResultsForQuoteFilters(_ filters: [QuoteListFilter]) -> Int {
+        handleLuggageAndPassengerFilters(filters)
         guard let quotes = fetchedQuotes else { return 0 }
         return quoteFilter.filter(quotes.all, using: filters).count
     }
+    
+    func handleLuggageAndPassengerFilters(_ filters: [QuoteListFilter]) {
```

Comment:
Event if this method could stay here (and it can't), it should be marked as `private`. 

`Handle` word is not very good as well, since the method is, in fact, updating local variables using new filters.